pkg/apis/velero/v1/backup.go

```diff
@@ -312,6 +312,12 @@ type BackupProgress struct {
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.phase",description="Backup status such as InProgress/Completed"
+// +kubebuilder:printcolumn:name="Errors",type="integer",JSONPath=".status.errors",description="Count of all error messages that were generated during execution of this backup"
+// +kubebuilder:printcolumn:name="Warnings",type="integer",JSONPath=".status.warnings",description="Count of all warning messages that were generated during execution of this backup"
+// +kubebuilder:printcolumn:name="Created",type="date",JSONPath=".status.startTimestamp",description="Time when this backup was started"
```

Comment:
suggest: `Start Time`

Reply:
I opted for the "Created" title to match the equivalent column from `velero get backups`. Same with "Age" to match the final column in standard `kubectl get` output (we already had that field/column for BSLs). Is consistency with those important?

Reply:
+1 for being consistent with the columns for backups and using `Created` .

Reply:
Agreed, I would prefer consistency.

Reply:
I think I would prefer to match the current `velero get backups` output, since that's established and updating it _may_ break user scripts, so `Created` works for me.

That does, however, put us in an awkward spot since I think all other `.status.startTimestamp` fields should match, due to the existence of the `creationTimestamp` field. Ideally, we'd have clean mappings between the field name in the YAML and printer column, but I think due to historical reasons we're a little stuck.

So my vote is this mapping:

Print `.status.startTimestamp` under `Created`
Print `.metadata.creationTimestamp` under `Age`

Reply:
Sorry, I'm not sure I understand what you mean by:
> That does, however, put us in an awkward spot since I think all other .status.startTimestamp fields should match, due to the existence of the creationTimestamp field.

As for the mapping you'd prefer, that is what is in this file. Am I missing something?

Reply:
The awkward spot I was referring to is that we have `creationTimestamp` that isn't mapped to the `Created` display name. It's probably not a big deal except for people looking at the code.

It looks like there's 2 instances where `status.startTimestamp` is mapped to `Start Time`, rather than `Created`:

 * https://github.com/vmware-tanzu/velero/pull/2881/files#diff-2882940cdbbb4c1142f4b9468c593dc0R38
 * https://github.com/vmware-tanzu/velero/pull/2881/files#diff-315e9e294c9cc0ab062afc00990fa0aaR18

Reply:
Ah, I see. Thanks for the explanation!

I used `Started` as those column names which matches the output from `velero get restores`... another inconsistency :(

Reply:
Gotcha - yeah, that was our past mistake.

I think we could put that on the docket for v2.0 to update those `velero` commands so we don't break any scripts.

Reply:
Created #2989 to address this.